src/plugins/vis_augmenter/public/actions/saved_object_delete_action.ts

```diff
@@ -45,12 +45,16 @@ export class SavedObjectDeleteAction implements Action<SavedObjectDeleteContext>
       throw new IncompatibleActionError();
     }
 
-    const loader = getSavedAugmentVisLoader();
-    const allAugmentVisObjs = await getAllAugmentVisSavedObjs(loader);
-    const augmentVisIdsToDelete = allAugmentVisObjs
-      .filter((augmentVisObj) => augmentVisObj.visId === savedObjectId)
-      .map((augmentVisObj) => augmentVisObj.id as string);
-
-    if (!isEmpty(augmentVisIdsToDelete)) loader.delete(augmentVisIdsToDelete);
+    try {
+      const loader = getSavedAugmentVisLoader();
+      const augmentVisObjs = await getAugmentVisSavedObjs(savedObjectId, loader);
+      const augmentVisIdsToDelete = augmentVisObjs.map(
+        (augmentVisObj) => augmentVisObj.id as string
+      );
+
+      if (!isEmpty(augmentVisIdsToDelete)) loader.delete(augmentVisIdsToDelete);
+      // silently fail. this is because this is doing extra cleanup on objects unrelated
+      // to the user flow so we dont want to add confusing log lines or error toasts
+    } catch (e) {} // eslint-disable-line no-empty
```

Comment:
You can at least log this error to the console.

Reply:
Added